docs/dev/workflow-code-paths.rst

```diff
@@ -25,9 +25,9 @@ The ``nrnivmodl`` executable translates the ``MOD`` files from the ``NMODL`` lan
 Circuit creation
 ^^^^^^^^^^^^^^^^
 
-During this part of the simulation the segments, sections and their connections are created. All these are created based on the circuit which is defined by the users in HOC or Python scripts with the respective API.
+During this part of the simulation the segments, sections and their connections are created. All these are created based on the circuit which is defined by the users in Python or HOC scripts with the respective API.
 
-See: :ref:`_hoc_topology` and :ref:`_topology`.
+See: :ref:`_topology` and :ref:`_hoc_topology`.
```

Comment:
FYI these aren't linking properly in https://nrn.readthedocs.io/en/magkanar-docs/dev/workflow-code-paths.html#circuit-creation, I think you need to remove the leading `_`